### ISSUE TITLE
API Feature Request: Add block / unblock to relations endpoint

### ISSUE THREAD

**Issue by LennieCodes** [lichess-org/api#197]
[profile relations](https://lichess.org/api#tag/Relations) endpoint, currently does not support unblocking / blocking a player. It would be nice to have those endpoints added to the relations API.

**Comment by ornicar** [comment]
What is your use case, why do you need it?

**Comment by johndoknjas** [comment]
@ornicar One use case for me is that I was interested in what percentage of players I block get banned. This is to get an idea of roughly how accurate my suspicions tend to be of cheating.
